src/containers.h

```diff
@@ -0,0 +1,51 @@
+/* Copyright (C) 2021 Open Information Security Foundation
+ *
+ * You can copy, redistribute or modify this Program under the terms of
+ * the GNU General Public License version 2 as published by the Free
+ * Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * version 2 along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301, USA.
+ */
+
+#ifndef __CONTAINERS_H__
+#define __CONTAINERS_H__
+
+void ContainersInit(void);
+void ContainersDestroy(void);
+uint32_t ContainersTimeoutHash(struct timeval *ts);
+
+void *ContainerUrlRangeGet(const uint8_t *key, size_t keylen, struct timeval *ts);
+
+typedef struct RangeContainer {
```

Comment:
please add docs to the fields in these structs

Reply:
ok